HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -155,4 +174,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     # add call to action function in proper order: newest last!
     # process = customiseFor12718(process)
 
+    process = customiseFor37180(process)
```

Comment:
```suggestion
```
Since the 4 HLT plugins all use `fillDescriptions`, and the customisation is not changing the default values of the parameters, I don't think this HLT customisation is needed in the first place (if the parameters are missing in python config, they will be set to default by fillDesc).

Reply:
You're right, seems not be necessary after all. I removed the customization.